src/main/java/org/opensearch/security/dlic/rest/api/AuthTokenProcessorAction.java

```diff
@@ -40,13 +40,16 @@
 import java.nio.file.Path;
 import java.util.Collections;
 import java.util.List;
+
 import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
 
 public class AuthTokenProcessorAction extends AbstractApiAction {
 	private static final List<Route> routes = addRoutesPrefix(Collections.singletonList(
 			new Route(Method.POST, "/authtoken")
 	));
 
+	public static final String API_AUTHTOKEN_SUFFIX = "api/authtoken";
```

Comment:
The const is defined but it is not used in this class, so it does not seems to be the right place to define it. Either define `public static final String AUTHTOKEN_SUFFIX = "/authtoken";` and replace all occurrences of `"/authtoken"` with AUTHTOKEN_SUFFIX (preferrable) or define `API_AUTHTOKEN_SUFFIX` where it is actually used.